gen/testdata/thrift/structs.thrift

```diff
@@ -74,4 +76,26 @@ struct Node {
     2: optional List next
 }
 
-// TODO: Default values
+
+//////////////////////////////////////////////////////////////////////////////
+// Default values
+
+struct DefaultsStruct {
```

Comment:
You copying snippets from ThriftTest.thrift? Or just writing them by hand as needed?